packages/composables/src/composables/useBilling/index.ts

```diff
@@ -3,23 +3,34 @@ import {
   useBillingFactory,
```

Comment:
If the logic for billing is related to cart at the backend maybe think about extending cart composable instead of covering useBilling. This approach will give you the ability to have cart value available out of the box.

Reply:
Shipping, Billing and Shipping provided is all related to cart but I thought that it would be better to have it separate as it requires different mutations/queries to be performed.